.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+**Notes**
+You can delete everything from the horizontal rule up before submitting your issue.
+- If you plan on picking up this issue immediately, go ahead and assign yourself now.
+- Label the issue appropiately, see the [Contributing Guide](https://github.com/aturingmachine/hockey-elo-tracker/blob/master/.github/CONTRIBUTING.MD) for help.
+- Make sure to add it to the `Elo Tracker` project using the sidebar to the left
+- If it involves the MVP API or Web, add it to that milestone
```

Comment:
Since you are adding periods every where else you should get these two little guys at the end 'Make sure ...' and 'If it involves...'